SPECS/kernel-hci/kernel-hci.spec

```diff
@@ -44,6 +44,11 @@ Patch22:        0023-net-mlx5-Bridge-rename-filter-fg-to-vlan_filter.patch
 Patch23:        0024-net-mlx5-Bridge-extract-VLAN-push-pop-actions-creati.patch
 Patch24:        0025-net-mlx5-Bridge-implement-infrastructure-for-VLAN-pr.patch
 Patch25:        0026-net-mlx5-Bridge-implement-QinQ-support.patch
+Patch26:        001-net-ice-core-netdevlink.patch
```

Comment:
Can you rename these patch files to `0027-net-ice-core-netdevlink.patch`, etc, so they are sorted properly in a `ls` output on the command line?